php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -500,7 +500,7 @@ class NDB_BVL_Instrument extends NDB_Page
 
                 // the form WAS submitted but validate() failed...
                 // main error message
-                $this->form->addElement('static', 'mainError');
+                $this->form->addElement('static', 'mainError', '');
```

Comment:
yeah i was thinking the same thing. that would be the better solution since i don't think forgetting to supply a label should cause everything to break but i wasn't 100% sure